reporters/index.js

```diff
@@ -171,8 +173,18 @@ async function Poll(callback) {
       ? parseInt(process.env.PERIOD_REPORT_LENGTH)
       : 24 * 60 * 60;
 
+    // Overrides the Uniswap pair that we want to query trade data for. The assumption is that the GlobalSummaryReporter fetches
+    // data for only one Uniswap pair, and the default pair tokens are the emp-synthetic-token and the emp-collateral-token.
+    const uniswapPairOverride = {
```

Comment:
Reference for https://github.com/UMAprotocol/protocol/pull/1733/files#r452888913

I thought that `index.js` was the correct place to hardcode the override Uniswap pair object. I also decided to map the EMP address to a **contract** instead of an address. This allows the `GlobalSummaryReporter` to access GET methods like `.symbol()` easier.

@mrice32

Reply:
That SGTM.

Only downside of mapping to a contract is that the `.at()` _may_ make a network call (to check whether there's code at that address, not 100% sure though), meaning that if we had a lot of these it may take a long time to initialize this map. That really only matters if we have a lot of stuff in this map, though. If we had like ~10+ members in this map, we'd probably want to parallelize using `Promise.all()`. If we had ~100, we'd probably just want to just map to addresses and initialize the object lazily once we pick one.

LGTM!

Reply:
Ah this is a great point actually. I'll add a comment but agree that with one object for now this is not a problem